packages/module/build.js

```diff
@@ -2,7 +2,7 @@
 const StyleDictionary = require('style-dictionary');
 
 const build = (selector) => {
-  const { fileHeader, formattedVariables } = StyleDictionary.formatHelpers;
+  const { fileHeader, formattedVariables, sortByName, sortByReference } = StyleDictionary.formatHelpers;
```

Comment:
Assuming we don't need to sort by reference anymore...

```suggestion
  const { fileHeader, formattedVariables, sortByName } = StyleDictionary.formatHelpers;
```